whotracksme/data/Readme.md

```diff
@@ -11,49 +11,164 @@ The tracker database is provided in the `assets/trackerdb.sql` file. This is a d
  * `trackers`: Metadata on trackers: name, description, category, website and an optional link to a parent company.
  * `tracker_domains`: Table linking trackers to domain names.
 
-## Data
+This SQL database links third-party domains to trackers, which are then linked to unique companies operating those trackers. This is similar to [Disconnect's Tracker List](https://bit.ly/2ZylQRF) and [webXray's Domain Owner List](https://t.ly/EHuu). These parties are already categorized accordingly within the WhoTracks.me datasets below.
 
-The data is created by aggregating data about page loads at several different levels. These aggregated stats are published in the following files:
+## WhoTracks.me datasets
 
- * `domains.csv`: Top third-party domains seen tracking.
- * `trackers.csv`: Top trackers - this combines domains known be operated by the same tracker.
- * `companies.csv`: Top companies - aggregates the stats for trackers owned by the same company.
- * `sites.csv`: Stats for number of trackers seen on popular websites.
- * `site_trackers.csv`: Stats for each tracker on each site.
+WhoTracks.me datasets are provided monthly in the `assets//{month}/{country}/{file}.csv` format. A glimpse of the datasets is available in [Explorer](https://whotracks.me/explorer.html) section on the website.
+
+### Data collection
 
-We calculate a standard set of signals across all aggregations:
+Data was collected from May 2017 from users that used Cliqz browser extension. In Feb 2018, 70% of the data came from German users according to [this](https://t.ly/NQcD) blog post. Then in March 2018, users of Ghostry FireFox extension - and Ghostry extension available for other browsers (Safari, Chrome, Opera and Edge) from users that opted-in to *HumanWeb* data collection - were added to the dataset. This caused a slight decrease in the avg. no. of trackers in April 2018, since Ghostry users were blocking more trackers. This is explained in [this](https://bit.ly/2VIBndZ) and [this](https://t.ly/BoCm) blog posts.
```

Comment:
```suggestion
Data was collected from May 2017 from users that used Cliqz browser extension. In Feb 2018, 70% of the data came from German users according to [this](https://t.ly/NQcD) blog post. Then in March 2018, users of Ghostery Firefox extension - and Ghostery extension available for other browsers (Safari, Chrome, Opera and Edge) from users that opted-in to *HumanWeb* data collection - were added to the dataset. This caused a slight decrease in the avg. no. of trackers in April 2018, since Ghostery users were blocking more trackers. This is explained in [this](https://bit.ly/2VIBndZ) and [this](https://t.ly/BoCm) blog posts.
```